app/battacker-web/src/components/Header.tsx

```
@@ -0,0 +1,10 @@
+export function Header() {
+  return (
+    <div className="header">
+      <div>
+        <h1 className="title">Battacker</h1>
+        <p className="subtitle">// Browser Defense Resistance Testing System</p>
```

Biome (2.3.13)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)